BlueEastCode/bluerain-ui-interfaces
# fix(Slider component fixes):

## What I did
Fixed slider component onValueChange method
## How to test

It is not testable in this repo.  But i tested it with bluerain-plugin-material-ui slider component.
Is this testable with jest or storyshots?  NO

Are any of the existing tests are failing ? NO

Does this need an update to the documentation? NO

Does it have breaking changes ? NO

If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid  please review the changes and merge them.
